README.md

```diff
@@ -153,6 +153,11 @@ This repository is a [monorepo](https://trunkbaseddevelopment.com/monorepos/) ma
 
 We are currently only accepting bug fixes for Gatsby v1. No new features will be accepted.
 
+### License
```

Comment:
To follow the style guide as the rest of the read me:
```suggestion
## License
```
Adding an icon next to License.